backend/prompt_studio/prompt_studio/migrations/0007_alter_toolstudioprompt_enforce_type.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 4.2.1 on 2024-08-07 14:20
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("prompt_studio", "0006_alter_toolstudioprompt_prompt_key_and_more"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="toolstudioprompt",
+            name="enforce_type",
+            field=models.TextField(
+                blank=True,
+                choices=[
+                    ("Text", "Response sent as Text"),
+                    ("number", "Response sent as number"),
+                    ("email", "Response sent as email"),
+                    ("date", "Response sent as date"),
+                    ("boolean", "Response sent as boolean"),
+                    ("json", "Response sent as json"),
+                    ("table", "Response sent as table"),
+                ],
+                db_comment="Field to store the type in             which the response to be returned.",
```

Comment:
@harini-venkataraman Migration file was not updated after changes in model is made.

Reply:
@Deepak-Kesavan Are you refering to this [file](https://github.com/Zipstack/unstract/pull/564/files/c561db60ce66db839e236956a584ab7ac29b5c09#diff-289bb837434ab5d22428caf83826719b37d042825a5a9cfe6a34d10581b77468)?
This is up to date with the model.

Reply:
Got it. Looks like additional spaces in the db_comment is included in the model file itself.